### PR TITLE
Update git clone URL

### PR DESCRIPTION
**Pull request by qolop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ We use [Metalsmith](http://www.metalsmith.io/), a static generator in NodeJS.
 2. Now clone this repository:
 
     ```sh
-    $ git clone git://github.com/zenorocha/dracula-theme.git
+    $ git clone https://github.com/dracula/dracula-theme.git
```
Comment (nesl247) [inline README.md]: Any reason for the change in protocol?

Comment (qolop) [inline README.md]: I see this used more often (such as on GitHub repositories themselves), there is no difference to my knowledge.

**Comment by qolop** [comment]
HTTPS seems to be the standard, according to official documentation. Again, both work fine.  https://help.github.com/articles/cloning-a-repository/

**Comment by nesl247** [comment]
I've not used http or https in years for git. Not since 2fa came out for GitHub that is, so it always looks weird to me to see it.

**Comment by qolop** [comment]
Ah, that will do it. Thanks for the merge.

**Pull request merged**